lab3.py

```diff
@@ -0,0 +1,61 @@
+"""
+1. Znaleźć w internecie API ze źródłem danych o rynkach finansowych, przykłady:
```

Comment:
nie trzymaj treści w plikach z kodem. Jeśli poprawnie zrobiłaś forka mojego repo to na swoje repo możesz zaciągnąć zmiany i mieć w nim aktualne pliki z treściami. Nie będą one widoczne w pull requestach i nie będę musiał ich oglądać/odróżniać miejsc treści od kodu. Czasem w treściach zadań podaję kawałki kodu i mogłoby to byc mylące